code_examples/core_features/getting_started/03_fetch_did.ts

```diff
@@ -2,12 +2,15 @@ import type { ApiPromise } from '@polkadot/api'
 
 import * as Kilt from '@kiltprotocol/sdk-js'
 
+// TODO: Replace api.call.didApi with api.call.did once it's deployed on Spiritnet
 export async function main(api: ApiPromise): Promise<Kilt.DidUri | null> {
-  const johnDoeOwner = await api.query.web3Names.owner('john_doe')
+  const johnDoeOwner = await api.call.didApi.queryDidByW3n('john_doe')
```

Comment:
Should this be updated and the TODO above removed?

Reply:
Spiritnet does not have the new `call.did` deployed yet, while Peregrine does. So the TODO will be removed in a new small PR once the new runtime API makes it on Spiritnet.

Reply:
Note that the getting started is the only code snippet that uses Spiritnet for demo purposes. The rest uses Peregrine by default.

Reply:
Oh, that’s why…